apps/angular/projection/src/app/component/city-card/city-card.component.ts

```diff
@@ -1,13 +1,40 @@
-import { Component, OnInit } from '@angular/core';
+import { Component, OnInit, ViewEncapsulation } from '@angular/core';
+import { CardComponent } from '../../ui/card/card.component';
+import { CityStore } from '../../data-access/city.store';
+import { CardType } from '../../model/card.model';
+import { ListItemComponent } from '../../ui/list-item/list-item.component';
+import { FakeHttpService } from '../../data-access/fake-http.service';
+import { ListItemDirective } from '../../ui/list-item-directive/list-item.directive';
 
 @Component({
   selector: 'app-city-card',
-  template: 'TODO City',
+  template: `<app-card
+    [list]="cities()"
+    [type]="cardType"
+    customClass="bg-lightyellow">
+    <ng-template listItem let-rowItem>
+      <app-list-item [id]="rowItem.id" [type]="cardType">{{
+        rowItem.name
+      }}</app-list-item>
+    </ng-template>
+  </app-card>`,
   standalone: true,
-  imports: [],
+  imports: [CardComponent, ListItemComponent, ListItemDirective],
+  styles: [
+    `
+      .bg-lightyellow {
+        background-color: lightyellow;
+      }
+    `,
+  ],
+  encapsulation: ViewEncapsulation.None,
```

Comment:
In this case it's not necessary to set `ViewEncapsulation.None`, i'll always try to avoid this approach, because it can lead to undesirable behavior. If i had to do this, i would use the component selector to apply the styles only for this component and his childs.

`
app-city-card {
.bg-lightyellow {
background-color: lightyellow;
}
}
`

Since you do this challenge already, maybe checkout the other solutions from the community or the author's solution, to see how it can work without changing the view encapsulation.

Reply:
yes you should always avoid this encapsulation. This create weird side effect. If you use a generic css name, this css will now be available to your entire application. 
If you need to pass css down, use css variables.